.hintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "no-inline-styles:error"
+  ]
```

Comment:
- [ ] Kindly do not modify this file in the future 🙏🏽🙏🏽 Their essence is to help us adhere to a certain standard.